sqlglot/dialects/tsql.py

```diff
@@ -224,6 +224,12 @@ class TSQL(Dialect):
     class Tokenizer(tokens.Tokenizer):
         IDENTIFIERS = ['"', ("[", "]")]
 
+        QUOTES = [
+            (prefix + quote, quote) if prefix else quote
+            for quote in ["'", '"']
+            for prefix in ["", "n", "N"]
+        ]
```

Comment:
I think we have something similar for MySQL, but we handle it as an "introducer" instead. If it's the same concept, then we should probably change either dialect so that we maintain consistency.

Reply:
I implemented this the same like in bigquery.

Reply:
I'm not sure if this is correct after all, because you can't preserve the `N` to later transpile it correctly.

Reply:
unless you're ok losing the N